web/src/_themes/oxide/static/oxide.css

```
@@ -0,0 +1,586 @@
+  overflow: auto;
+  border-color: var(--bg-color-inv);
+  border-style: solid;
+  border-width: 0;
+  border-left-width: initial;
+  font-size: 0.888rem;
+  color: var(--font-color-inv);
+  text-align: left;
+  font-weight: normal;
+  margin: 0;
+  font-style: italic;
+  color: var(--font-color);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)